docs/operations/query-concurrency.md

```diff
@@ -0,0 +1,174 @@
+---
+id: query-concurrency
+title: Query concurrency
```

Comment:
I feel there could be a better title than this.. Would "Multi-workloads support" be better?